src/the-bureau/bureau-structure/index.html

```diff
@@ -31,8 +31,8 @@
 
         <div class="media org-chart_root">
             <img class="media_image media_image__150" src="/static/img/director-cordray-round-300x300.jpg">
-            <h1 class="h2 u-mb0">Richard Cordray</h1>
-            <h2 class="h4">Director</h2>
+            <p class="h2 u-mb0">Richard Cordray</p>
+            <h1 class="h4">Director</h1>
```

Comment:
The `h1` of this page should be "Bureau Structure" (even if it's visually hidden on standard browsers).